docs/_manpage.html

```diff
@@ -11,10 +11,11 @@
 #define CURL_URL docs/manpage.html
 
 #include "_menu.html"
+#include "docs/_option-menu.html"
```

Comment:
Template for option menu "_option-menu.html", and the generated options from perl script "manoptionsdump.gen".